hq/app/config/Config.scala

```diff
@@ -13,6 +14,8 @@ import scala.util.Try
 
 
 object Config {
+  val region: Regions = Regions.EU_WEST_1
```

Comment:
So this change is an interesting one!

It was originally hard coded in the AppComponents, which is the "edge" of the program that starts our webserver. In truth, this shouldn't be hard-coded at all, we should be reading it from the AWS metadata somehow. Moving it here is a good choice while it is static, but is a step backwards in terms of us being able to un-hard-code it in future. As I mention elsewhere in this review, if we're doing a lookup we'll want to keep the IO "at the edge" rather than have the instantiation of our Config object at load time cause trouble when it starts running off to the network to fill in the vals.

I don't feel strongly one way or the other with this change here - it is hard coded now so this looks great. I just wanted to share the thinking about why it was where it was :-)